sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpEventHubConsumer.cs

```diff
@@ -42,6 +45,26 @@ internal class AmqpEventHubConsumer : TransportEventHubConsumer
         ///
         public override bool Closed => _closed;
 
+        /// <summary>
```

Comment:
Why not have these as private readonly fields instead of making them properties?

Reply:
Because our style checker has been.... unreliable with respect to allowing `readonly` to be treated as `const that we initialize at construction.`   This is a concession to our tooling, though it's a toss up either way.